src/main/java/com/google/sps/user/repository/UserRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.google.sps.user.repository;
+
+import com.google.sps.user.User;
+import org.jetbrains.annotations.Nullable;
+import java.util.HashMap;
+
+/** Handles the storage of User data. */
+public interface UserRepository {
+    public void saveUser(User user);
+
+    @Nullable
+    public User getUser(String id);
```

Comment:
Maybe add some docs. At least explain when this returns null.

Reply:
Added.